pkgs/development/python-modules/pydantic-core/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, cargo
+, rustPlatform
+, rustc
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "pydantic-core";
+  version = "2.1.3";
```

Comment:
The commit for this contains an extra v in the version number

Reply:
fixed, thx